AntennaTracker/wscript

```diff
@@ -11,6 +11,7 @@ def build(bld):
             'AP_Arming',
             'AP_RCMapper',
             'AP_OSD',
+            'AP_MSP',
```

Comment:
Add to here instead: https://github.com/ardupilot/ardupilot/blob/master/Tools/ardupilotwaf/ardupilotwaf.py#L101

Reply:
..and remove it from each vehicle wscript?